package.json

```
@@ -152,7 +152,7 @@
     "react-native-linear-gradient": "^2.8.1",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "2.0.27",
+    "react-native-onyx": "^2.0.35",
```

Let's please not use the `^` syntax - just pin to specific version so it's clear and we don't ever upgrade unintentionally.

```suggestion
    "react-native-onyx": "2.0.35",
```